src/core/context/CookiesContext.tsx

```diff
@@ -20,7 +20,19 @@ export type CookiesContextValues = {
 const CookiesContextTracking = createContext<CookiesContextValues>({} as CookiesContextValues);
 const useCookiesContextTracking = () => useContext(CookiesContextTracking);
 
-const CookiesProviderTracking: React.FC<React.PropsWithChildren> = ({ children }) => {
+interface Props {
+  timestampTracking: string;
+  themeTracking: string;
+  analyticsTracking: string;
+  children: ReactNode;
```

Comment:
This is not needed as you're using `React.PropsWithChildren & props` in the type declaration if the provider. A better solution if you use an `interface` would be:

```js
interface Props extends React.PropsWithChildren {
  // ...
}
```

Reply:
ok